python/python/glide/async_commands/core.py

```diff
@@ -4629,3 +4642,28 @@ async def srandmember_count(self, key: str, count: int) -> List[str]:
             List[str],
             await self._execute_command(RequestType.SRandMember, [key, str(count)]),
         )
+
+    async def flushall(self, flush_mode: Optional[FlushMode] = None) -> TOK:
+        """
+        Delete all the keys of all the existing databases, not just the currently selected one. This command never fails.
+
+        See https://valkey.io/commands/flushall for more details.
+
+        Args:
+            flush_mode (Optional[FlushMode]): An optional argument that dictates the flushing mode explicitly.
```

Comment:
Please use the existing documentation for Java: 
https://github.com/aws/glide-for-redis/blob/main/java/client/src/main/java/glide/api/commands/ServerManagementCommands.java#L155-L181